worlds/dkc3/Locations.py

```diff
@@ -321,13 +322,13 @@ def __init__(self, player: int, name: str = '', address: int = None, parent=None
 location_table = {}
 
 
-def setup_locations(world, player: int):
+def setup_locations(world: World):
     location_table = {**level_location_table, **boss_location_table, **secret_cave_location_table}
 
-    if False:#world.include_trade_sequence[player].value:
+    if False:#world.options.include_trade_sequence:
```

Comment:
```suggestion
    if False:  # world.options.include_trade_sequence:
```
If this comment is to stay, might as well make it pep8? :P